### PR TITLE
Add data-origin to status bar items for better observability

### PR DESCRIPTION
**Pull request by aleksfront**
It's now easier to use `[data-origin=myextension]` selectors to find status bar elements with `document.querySelector()` or similar.

data-origin DOM example

### CODE REVIEW

**Review by Nokel81 — DISMISSED** [review]

File: src/renderer/components/status-bar/status-bar-item-registrator.injectable.tsx [hunk 1]
```diff
@@ -74,6 +74,7 @@ const toItemInjectableFor = (extension: LensRendererExtension, getRandomId: () =
       id,
 
       instantiate: () => ({
+        id: extension.sanitizedExtensionId,
```
Comment (jansav) [inline src/renderer/components/status-bar/status-bar-item-registrator.injectable.tsx]: There's something wrong here. This makes all of the status bar items from extension to have same ID?

Comment (jansav) [reply src/renderer/components/status-bar/status-bar-item-registrator.injectable.tsx]: If that's the intended behaviour, then ID is not correct term for it.

**Review by jansav — CHANGES_REQUESTED** [review]
Left single comment

File: src/renderer/components/status-bar/status-bar-item-registrator.injectable.tsx (hunk 1 above)
Comment (aleksfront) [reply src/renderer/components/status-bar/status-bar-item-registrator.injectable.tsx]: Yes, indeed. It should be called an `origin` not `id`.

**Review by Nokel81 — APPROVED** [review]

File: src/renderer/components/status-bar/status-bar-items.injectable.tsx [hunk 2]
```diff
@@ -10,9 +10,14 @@ import type { StatusBarItem } from "./status-bar-item-injection-token";
 import { statusBarItemInjectionToken } from "./status-bar-item-injection-token";
 import { computedInjectManyInjectable } from "@ogre-tools/injectable-extension-for-mobx";
 
+interface StatusItem {
+  origin?: string;
+  component: React.ComponentType<StatusBarItemProps>;
```
Comment (jansav) [inline src/renderer/components/status-bar/status-bar-items.injectable.tsx:15]: Small things: I would expect this to be `Component` as it's a React.Component.

File: src/renderer/components/status-bar/status-bar.tsx [hunk 3]
```diff
@@ -22,18 +22,24 @@ const NonInjectedStatusBar = observer(({ items }: Dependencies & StatusBarProps)
   const { left, right } = items.get();
 
   return (
-    <div className={styles.StatusBar}>
+    <div className={styles.StatusBar} data-testid="status-bar">
       <div className={styles.leftSide} data-testid="status-bar-left">
         {left.map((Item, index) => (
```
Comment (jansav) [inline src/renderer/components/status-bar/status-bar.tsx:27]: Same here, `Item` was capitalized to describe that it is a React.Component (or a class). However, now it's not anymore.

**Review by jansav — APPROVED** [review]

**Pull request merged**